### PR TITLE
Multiple debugging updates to serial interface

### LINKED ISSUE #6: Implement proper serial commands handling
Need to handle a few commands from serial port for debug, testing and configuration. Need to implement the system as a whole, and a few necessary commands:

* [ ] Health report
* [x] Reboot
* [x] Factory reset
* [x] Set up WiFi credentials
* [x] Getting firmware config

### PR DESCRIPTION
**Pull request by Eirenliel**
Fixes #6 and #215

Adds some minor fixes to logging for testing and debugging purposes.

### CODE REVIEW

File: src/status/StatusManager.cpp [hunk 1]
```diff
@@ -34,5 +34,45 @@ namespace SlimeVR
...
+        String StatusManager::getStatusString()
+        {
+            if(m_Status == 0)
+            {
+                return "NONE";
+            }
+            String statusString;
+            if(hasStatus(Status::LOADING))
+            {
+                statusString = "LOADING";
```
Comment (TheDevMinerTV) [inline src/status/StatusManager.cpp:47]: not sure if we should rather just send a bitmask from the enums 🤔

**Comment by TheDevMinerTV** [comment]
Also compiler error: [`src/network/wifihandler.h:38:39: error: 'uint8_t WiFiNetwork::getWiFiState()' should have been declared inside 'WiFiNetwork'`](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/actions/runs/3850806511/jobs/6561363180#step:5:347)

File: src/network/wifihandler.h [hunk 2]
```diff
@@ -35,6 +35,7 @@ namespace WiFiNetwork {
     void upkeep();
     void setWiFiCredentials(const char * SSID, const char * pass);
     IPAddress getAddress();
+    uint8_t WiFiNetwork::getWiFiState();
```
Comment (TheDevMinerTV) [inline src/network/wifihandler.h:38]:
```suggestion
    uint8_t getWiFiState();
```

File: src/status/StatusManager.cpp (hunk 1 above)
Comment (Eirenliel) [reply src/status/StatusManager.cpp]: Yeah, I changeed my mind and will return bitmask.

**Review by TheDevMinerTV — APPROVED** [review]

**Pull request merged**